venues/ICLR.cc/2019/Conference/python/iclr19.py

```diff
@@ -336,21 +336,16 @@
         'writers': {'values-regex': '~.*'},
         'signatures': {'values-regex': '~.*'},
         'content': {
-            'title': {
```

Comment:
All the notes need a title, otherwise we will render a 'No Title' string is that ok?

Reply:
mohit and I talked about this, it seems to be ok